server/dba/migrations/10-tempOutfitter.js

```diff
@@ -0,0 +1,64 @@
+      applicantInfoPrimaryMailingAddress2: { type: Sequelize.STRING, field: 'appl_info_pri_mailing_address2' },
+      applicantInfoPrimaryMailingCity: { type: Sequelize.STRING, field: 'appl_info_pri_mailing_city', allowNull: false },
+      applicantInfoPrimaryMailingState: { type: Sequelize.STRING(2), field: 'appl_info_pri_mailing_state', allowNull: false },
+      applicantInfoPrimaryMailingZIP: { type: Sequelize.STRING(5), field: 'appl_info_pri_mailing_zip', allowNull: false },
+      applicantInfoSecondaryMailingAddress: { type: Sequelize.STRING, field: 'appl_info_sec_mailing_address' },
+      applicantInfoSecondaryMailingAddress2: { type: Sequelize.STRING, field: 'appl_info_sec_mailing_address2' },
+      applicantInfoSecondaryMailingCity: { type: Sequelize.STRING, field: 'appl_info_sec_mailing_city' },
+      applicantInfoSecondaryMailingState: { type: Sequelize.STRING(2), field: 'appl_info_sec_mailing_state' },
+      applicantInfoSecondaryMailingZIP: { type: Sequelize.STRING(5), field: 'appl_info_sec_mailing_zip' },
+      applicantInfoOrganizationName: { type: Sequelize.STRING, field: 'applicant_info_org_name' },
+      applicantInfoWebsite: { type: Sequelize.STRING, field: 'applicant_info_website' },
+      applicantInfoOrgType: { type: Sequelize.STRING, field: 'applicant_info_org_type', allowNull: false },
+      applicantInfoSecondaryFirstName: { type: Sequelize.STRING, field: 'applicant_info_sec_first_name' },
+      applicantInfoSecondaryLastName: { type: Sequelize.STRING, field: 'applicant_info_sec_last_name' },
+      type: { type: Sequelize.STRING, field: 'type', allowNull: false },
+      tempOutfitterFieldsIndividualCitizen: { type: Sequelize.BOOLEAN, field: 'temp_outfitter_indiv_citizen', allowNull: false },
+      tempOutfitterFieldsSmallBusiness: { type: Sequelize.BOOLEAN, field: 'temp_outfitter_small_business', allowNull: false },
+      tempOutfitterFieldsActivityDescription: { type: Sequelize.STRING, field: 'temp_outfitter_activity_desc' },
+      tempOutfitterFieldsAdvertisingUrl: { type: Sequelize.STRING, field: 'temp_outfitter_advertising_url' },
+      tempOutfitterFieldsAdvertisingDescription: { type: Sequelize.STRING, field: 'temp_outfitter_advertising_desc' },
+      tempOutfitterFieldsClientCharges: { type: Sequelize.STRING, field: 'temp_outfitter_client_charges' },
+      tempOutfitterFieldsExperienceList: { type: Sequelize.STRING, field: 'temp_outfitter_exp_list' },
+      createdAt: { type: Sequelize.DATE, defaultValue: Sequelize.NOW, allowNull: false, field: 'created' },
+      updatedAt: { type: Sequelize.DATE, defaultValue: Sequelize.NOW, allowNull: false, field: 'updated' },
+      status: { type: Sequelize.STRING, defaultValue: 'Received', field: 'status'},
+      signature: { type: Sequelize.STRING(3), field: 'signature', allowNull: false},
+      reasonForReturn: { type: Sequelize.STRING, field: 'reason_for_return'}
```

Comment:
just to check that all of these migrations will not need the db to be recreated after this? Per our conversation, about transactions?